### ISSUE TITLE
A11y_Azure Tools Storage Explorer_ Toolbar_ Toolbar controls_Screenreader: Screen reader is not announcing the “copied” information when user activates the copy button present on blob container toolbars

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6333]
**GitHub Tags:**
#A11yMAS; #A11yTCS; #Win11; #DesktopApp; #A11ySev3; #BM_AzureToolsStorageExplorer_Win32_Dec2022; #Azure Tools Storage Explorer; #FTP; #Screenreader; #WCAG1.3.1; #NVDA; #Element:Button; #STP; #JAWS; 

### Actual results:
Screen reader is not announcing the “copied” information when user activates the copy button present on blob container toolbars.

**similar issue also observed with JAWS**
### Expected results:
Screen reader should announce the “copied” information when user activates the copy button by pressing on enter.
### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0
NVDA version 2020.4
JAWS version 2021.205.53
### OS Details
Microsoft Windows 11 Enterprise
version 22H2 OS Build 22621.819
### Repro Steps:
1. Launch Storage Explorer
2. Tab to "Open Connect dialog" pane and Sign into Azure.
3. Tab to "Manage Accounts" pane and select the subscriptions you will use. Once selected, click on the “Apply" button.
4. Tab to "Blob Containers" of the selected subscription, right click and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Tab to "Upload" button and upload a blob/file by using “Upload Files" control.
6.  Turn on NVDA
7. Navigate to Toolbar and tab to Copy button and press on enter to activate
8. And check whether the screen reader is announcing the copied information when user activates the copy button or not.

### User Impact:
Screen reader user will not know about copied information If it’s not announced.
### Attachments:

https://user-images.githubusercontent.com/100552367/205639863-56f5e731-b515-41f5-b5f6-5c0c13cc431a.mp4

https://user-images.githubusercontent.com/100552367/205640925-4e20e7d2-74b0-432e-984f-5e487e99feff.mp4

**Comment by Jtith** [comment]
Verified in the 1.28.0 build issue is still reproducing 
[6333_Still repro.zip](https://github.com/microsoft/AzureStorageExplorer/files/10502458/6333_Still.repro.zip)

**Comment by Jtith** [comment]
This is a sev3 issue you can fix the bug or add it in your backlog

**Comment by Mana-swi** [comment]
#a11ydb-action-no-confirm;
